src/gui/transferlistwidget.h

```diff
@@ -117,9 +117,15 @@ protected slots:
     void setSelectedAutoTMMEnabled(bool enabled) const;
     void askNewCategoryForSelection();
     void saveSettings();
+#ifdef Q_OS_WIN
+    void taskbarChanged();
+#endif
 
 signals:
     void currentTorrentChanged(BitTorrent::TorrentHandle *const torrent);
+#ifdef Q_OS_WIN
+    void updateTaskbar(BitTorrent::TorrentHandle *const torrent);
```

Comment:
IMO `void updateTaskbar(const BitTorrent::TorrentHandle *torrent);` is better.

Reply:
As I said in my previous comments it's incorrect dependency here. So this signal shouldn't exist (at least in its current form/meaning).